schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/SchemaVersion.java

```diff
@@ -28,6 +29,7 @@
 
     private String description;
     private String schemaText;
+    private Byte initialState = SchemaVersionLifecycleStates.ENABLED.id();
```

Comment:
I made it explicit that it is the initial state of the version being added. I am fine with either `stateId` or `initialStateId`.